allennlp_models/lm/models/next_token_lm.py

```diff
@@ -117,7 +117,13 @@ def forward(  # type: ignore
         # Compute loss.
         if target_ids is not None:
             batch_size, vocab_size = target_logits.size()
-            targets = util.get_token_ids_from_text_field_tensors(target_ids).view(batch_size)
+            tmp = util.get_token_ids_from_text_field_tensors(target_ids)
+            # In some scenarios, target_ids might be a topk list of token ids (e.g. sorted by probabilities).
+            # Therefore, we need to make sure only one token per batch
+            # Assume: first token in each batch is the most desirable one (e.g. highest probability)
+            tmp = tmp[:, 0] if len(tmp.shape) == 2 else tmp
+            assert len(tmp.shape) <= 2
+            targets = tmp.view(batch_size)
```

Comment:
These changes are in https://github.com/allenai/allennlp-models/pull/213/files, right? If so, can you remove them from this PR?